torch/fx/experimental/proxy_tensor.py

```diff
@@ -1217,6 +1222,11 @@ def __torch_function__(
         return func(*args, **kwargs)
 
 
+_temp_remove_metadata_torch_function_mode = _make_temp_remove_mode_context_manager(
+    TorchFunctionMetadataMode
```

Comment:
@ydwu4 I'm confused because I thought we wanted to enable this mode inside the body of cond/while-loop?

Reply:
Yeah, we should enable these otherwise the subgraphs will lose the metadata.